src/Bonsai.Lsl/StreamInlet.cs

```diff
@@ -315,4 +321,40 @@ IObservable<Timestamped<TResult>> Generate<TResult>(Func<Native.StreamInlet, TRe
             });
         }
     }
+
+    /// <summary>
+    /// Options for post-processing of samples for an LSL stream inlet.
+    /// </summary>
+    public enum ProcessingFlags
```

Comment:
I would rename this enum to `ProcessingOptions` for consistency with LSL terminology.